plugins/kotlin/code-insight/intentions-k2/src/org/jetbrains/kotlin/idea/k2/codeinsight/intentions/ConvertToRawStringTemplateIntention.kt

```diff
@@ -43,5 +45,13 @@ internal class ConvertToRawStringTemplateIntention :
     }
 
     override fun isApplicableByPsi(element: KtBinaryExpression) =
-        element.descendantsOfType<KtStringTemplateExpression>().all { it.canBeConvertedToStringLiteral() }
+        element.descendantsOfType<KtStringTemplateExpression>().all { isApplicableWithSpaceBeforeLineBreak(it) }
```

Comment:
Can you just fix `canBeConvertedToStringLiteral` to support this case and not create a new string template with stripped spaces?